working/wildcards/feature-specification.md

```diff
@@ -317,37 +317,54 @@ class C {
 
 ### Super parameters
 
-An occurrence of `super._` as a declaration of a formal parameter in a
-constructor is a compile-time error. This error also occurs in the case
-where the super parameter has an explicitly declared type and/or default
-value.
+The positional formal parameter `super._` is still allowed in generative
+constructors.
 
-*`super._` is not an error everywhere: In a method body it could be an
-invocation of an inherited getter named `_`.*
+`super._` forwards the argument's value to the super constructor invocation.
 
 ```dart
-class B {
-  final _;
-  B(this._);
+class A {
+  final int _;
+  A(this._);
 }
 
-class C extends B {
-  C(super._); // Error.
+class B extends A {
+  B(super._); // OK.
 }
 ```
 
-*The desugared meaning of a super parameter includes a reference to the
-parameter in the initializer list of the enclosing constructor declaration,
-but such references are not possible when the parameter name is a
-wildcard.*
+Similar to `this._`, no parameter named `_` is bound. `_` can't be accessed in
```

Comment:
I'd say "no variable" or "no identifier" instead of "no parameter".

And I'd start the second sentence with a word instead of "\`_\`", maybe even:
> The value of the `super._` parameter can't be accessed ...